src/dotnet/Users.Service/UserProfilesBackend.cs

```diff
@@ -39,6 +40,28 @@ public UserProfilesBackend(IServiceProvider services) : base(services)
         return await ToUserProfile(user, cancellationToken).ConfigureAwait(false);
     }
 
+
+
+    // [CommandHandler]
+    public virtual async Task UpdateStatus(
+        IUserProfilesBackend.UpdateStatusCommand command,
+        CancellationToken cancellationToken = default)
+    {
+        if (Computed.IsInvalidating()) {
+            _ = AuthBackend.GetUser(command.UserId, default);
```

Comment:
Yep, that's right, and this thing also invalidates `IUserProfiles.Get`.

Reply:
done, but to `IUserProfilesBackend.Get`